src/instructlab/cli/model/download.py

```diff
@@ -57,9 +57,9 @@
 )
 @click.option(
     "--hf-token",
-    default="",
+    default=read_hf_token_from_file(),
```

Comment:
@cdoern will this work with our config scheme?

Reply:
seems not a good way to use in `default`, I updated it to `hf_token=read_hf_token_from_file(hf_token)`.